src/foam/u2/property/MDCheckBox.js

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
Can we share more code with regular CheckBox by either extending it or having both extend an UnstyledCheckBox or AbstractCheckBox?